src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -287,7 +299,7 @@ export const SpendReport = (props: SpendReportProps) => {
         {
           style: {
             display: 'grid',
-            gridTemplateColumns: 'repeat(3, minmax(max-content, 1fr))',
+            gridTemplateColumns: `repeat(${props.cloudPlatform === 'GCP' ? 3 : 4}, minmax(max-content, 1fr))`,
```

Comment:
Perhaps define the array of card names (used below at 345) as variables so you can use the length of the arrays here?

Reply:
And actually, that variable value could depend on the the `cloudPlatform`, which would avoid the need to check the `cloudPlatform` here and at line 345.